crates/rpc/src/handlers/root.rs

```diff
@@ -0,0 +1,21 @@
+use ream_storage::db::ReamDB;
+use tree_hash::TreeHash;
+use warp::{
+    http::status::StatusCode,
+    reject::Rejection,
+    reply::{Reply, with_status},
+};
+
+use super::{BeaconResponse, state::get_state_from_id};
+use crate::types::id::ID;
+
+pub async fn get_root(state_id: ID, db: ReamDB) -> Result<impl Reply, Rejection> {
+    let state = get_state_from_id(state_id, &db).await?;
+
+    let state_root = state.tree_hash_root();
+
+    Ok(with_status(
+        BeaconResponse::json(state_root),
```

Comment:
```suggestion
        BeaconResponse::json(RootResponse::new(state_root)),
```

You need to put the state_root into a struct with a field called `root` for this to serial right

```rust
#[derive(Serialize, Deserialize)]
struct RootResponse {
    root: B256,
}

impl RootResponse {
    pub fn new(root: B256) -> Self {
        Self {
            root
        }
    }
}
```

You can use this for that ^, please put this RootResponse struct in `mod.rs` as it has to be used for the block root api as well